valve/valve.py

```diff
@@ -1138,6 +1130,28 @@ def in_set(table_details, args, value):
     return False, f"'{value}' must be in: " + ", ".join(allowed)
 
 
+def not_any_of(config, args, value, lookup_value=None):
+    """Method for the VALVE 'not' function.
+
+    :param config: valve config dictionary
+    :param args: arguments provided to not
+    :param value: value to run not on
+    :param lookup_value: value required for 'lookup' when lookup is in the arguments
+    :return: True if value does not pass any condition in the arguments
+    """
+    conditions = []
+    for arg in args:
+        if not meets_condition(config, arg, "", value, when_value=lookup_value)[0]:
+            # As long as one is "NOT" met, this passes
```

Comment:
This doesn't seem right. If my rule is `not(blank, prefix)` then I would interpret it as `not(blank) and not(prefix)`. So I pass a prefix I expect the condition to fail. But this code sees that the prefix is `not(blank)` and returns success.

Reply:
You're right, I fixed this.